src/wallet/receive.cpp

```diff
@@ -265,7 +265,7 @@ void CachedTxGetAmounts(const CWallet& wallet, const CWalletTx& wtx,
             address = CNoDestination();
         }
 
-        COutputEntry output = {address, txout.nValue, (int)i};
+        COutputEntry output = {address, std::move(txout.scriptPubKey), txout.nValue, (int)i};
```

Comment:
Oh, yeah, i wanted to revisit this but forgot and pushed.. It's invalid since the caller will use the `scriptPubKey` (for instance in `WalletTxToJSON`).